webrender/src/internal_types.rs

```diff
@@ -45,6 +45,8 @@ pub enum SourceTexture {
     External(ExternalImageData),
     CacheA8,
     CacheRGBA8,
+    RenderTaskCacheA8(usize),
```

Comment:
We could use a newtype here to give a bit of type safety, and more importantly, self-document what these fields are. For example:

```
struct RenderPassIndex(usize);

...
RenderTaskCacheA8(RenderPassIndex),
...
```

Or something like that?